pybossa/view/projects.py

```diff
@@ -517,6 +518,57 @@ def clone(short_name):
         project=project_sanitized
     ))
 
+@blueprint.route('/<short_name>/tasks/taskpresenterimageupload', methods=['GET', 'POST'])
+@login_required
+@admin_or_subadmin_required
+@csrf.exempt
+def upload_task_guidelines_image(short_name):
+    error = False
+    project = project_by_shortname(short_name)
+
+    disable_editor = (not current_user.admin and
+                      current_app.config.get(
+                          'DISABLE_TASK_PRESENTER_EDITOR'))
+
+    is_admin_or_owner = (
+        current_user.admin or
+        (project.owner_id == current_user.id or
+            current_user.id in project.owners_ids))
+
+    if disable_editor:
+        flash(gettext('Task presenter editor disabled!'), 'error')
+        error = True
+    elif not is_admin_or_owner:
+        flash(gettext('Ooops! Only project owners can upload files.'), 'error')
+        error = True
+
+    imgurls = []
+    large_file = False
+    for file in request.files.getlist("image"):
+        file_size_mb = file.seek(0, os.SEEK_END) / 1024 / 1024
+        file.seek(0, os.SEEK_SET)
+        file.filename = secure_filename(file.filename)
+        if file_size_mb < current_app.config.get('MAX_IMAGE_UPLOAD_SIZE_MB'):
```

Comment:
It would be a good idea to also include a default value in case a dev does not have this setting.

```python
current_app.config.get('MAX_IMAGE_UPLOAD_SIZE_MB', 5)
```